packages/driver/test/cypress/integration/commands/actions/click_spec.js

```diff
@@ -378,6 +378,49 @@ describe('src/cy/commands/actions/click', () => {
       cy.getAll('div', 'pointerover pointerenter pointerdown mousedown pointerup mouseup click').each(shouldBeCalled)
     })
 
+    it('events when element removed on pointerdown', () => {
```

Comment:
@Bkucera why is this code here? this feels like code from the other `click` PR and not type related?

Reply:
snuck in during merge, fixed